packages/react/src/components/Slider/Slider.js

```diff
@@ -14,6 +14,8 @@ import throttle from 'lodash.throttle';
 import * as keys from '../../internal/keyboard/keys';
 import { matches } from '../../internal/keyboard/match';
 import deprecate from '../../prop-types/deprecate';
+import requiredIfValueExists from '../../prop-types/requiredIfValueExists';
+import { useControlledStateWithValue } from '../../internal/FeatureFlags';
```

Comment:
Do we need to rope this behind a feature flag? Thought this was potentially behavior from a previous Slider version but could definitely be misunderstanding 👀

Reply:
no it's probably not necessary so I removed it